.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Did you mean to include this file?

Reply:
Yeah, it's a generic VSCode debugging setup for windows.